pallets/creditcoin/src/tests.rs

```diff
@@ -2755,7 +2827,7 @@ fn register_funding_transfer_should_error_when_not_deal_order_not_found() {
 		let tx = "0xabcabcabc";
 
 		assert_noop!(
-			Creditcoin::register_funding_transfer(
+			Creditcoin::register_funding_transfer_legacy(
```

Comment:
this is causing changes in code coverage reports. IMO we should be exercising the new extrinsics and either copy or parametrize the tests to exercise the old extrinsics as well.